content/en/deployment_gates/setup.md

```diff
@@ -115,13 +122,39 @@ curl -X POST "https://api.{{< region-param key="dd_site" >}}/api/unstable/deploy
 
 **Note**: A 404 HTTP response can be because the gate was not found, or because the gate was found but has no rules.
 
-If a 200 HTTP status code is returned, the response is in the following format:
+If the gate evaluation was successfully started, a 202 HTTP status code is returned. The response is in the following format:
 
 ```json
 {
    "data": {
        "id": "<random_response_uuid>",
-       "type": "deployment_gates_evaluation_response",
+        "type": "deployment_gates_evaluation_response",
+        "attributes": {
+            "evaluation_id": "e9d2f04f-4f4b-494b-86e5-52f03e10c8e9"
+        }
+    }
+}
+```
+
+The field `data.attributes.evaluation_id` contains the unique identifier for this gate evaluation.
+
+An additional API endpoint allows to fetch the gate evaluation status. You can poll this endpoint using the Gate Evaluation ID to fetch the Gate Evaluation status:
+
+```bash
+curl -X GET "https://api.{{< region-param key="dd_site" >}}/api/unstable/deployments/gates/evaluation/e9d2f04f-4f4b-494b-86e5-52f03e10c8e9" \
```

Comment:
We should probably write it like:

```suggestion
curl -X GET "https://api.{{< region-param key="dd_site" >}}/api/unstable/deployments/gates/evaluation/<evaluation_id>" \
```

and mention to replace <evaluation_id> with the ID returned from the previous endpoint (in our example, `e9d2f04f-4f4b-494b-86e5-52f03e10c8e9`)